### ISSUE TITLE
Link from Conservation Analysis table in Variant Type/Missense Tab

### ISSUE THREAD

**Issue by wrightmw** [ClinGen/clincoded#1881]
Provide link from Conservation Analysis table in Variant Type/Missense Tab to the comparison view in UCSC.

Here is a link from a variant as an example:

http://genome.ucsc.edu/cgi-bin/hgTracks?clade=mammal&org=Human&db=hg19&position=chr7:151257666-151257726

The pertinent points are:
1. specifying genome
so in this case hg19
2. and then for that genomic range, -30nt and +30nt from the variant’s genomic position
3. automatically show mammals (look into this!)

**Comment by wrightmw** [comment]
@markmandell This is great!
Looking at ClinVar VariationID:17000 as an example. 

And clicking on your "UCSC Genome browser" link in the table header:

![Screen Shot 2019-07-16 at 8 52 39 PM](https://user-images.githubusercontent.com/15131169/61345897-cb464f00-a80b-11e9-89af-be4abc53fda4.png)

This is the view I get to from that entry (https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&lastVirtModeType=default&lastVirtModeExtraState=&virtModeType=default&virtMode=0&nonVirtPosition=&position=chr13%3A20763590%2D20763650&hgsid=740048535_MIjmpi8MJpqRQC4wJVzGiibjnNUT):
![Screen Shot 2019-07-16 at 8 51 18 PM](https://user-images.githubusercontent.com/15131169/61345839-8de1c180-a80b-11e9-81bc-30d4c8eea9d2.png)

@sharriso @marinadistefano Is this the link/view you had in mind? Is the "UCSC Genome Browser" linking text ok, or would you prefer something more descriptive...such as "View UCSC Genome Browser Multiz alignments"?

**Comment by marinadistefano** [comment]
Thanks Matt and Mark. This looks good to me. I agree that we might want something a bit more specific like “view position in UCSC Genome Browser” or, if that’s too wordy, what you had suggested is fine.

I have all of my browsers defaulted to this, but does think link out automatically display the extended list of species or will someone have to set that on their browser?

**Comment by wrightmw** [comment]
Thanks @marinadistefano, “view position in UCSC Genome Browser” looks good to me!
The default is just that small set species shown when accessing via a URL. A user has to manually change the default to view all species, but when they do save any changes then their browser should be defaulted to that full view when they return via the same link.

**Comment by markmandell** [comment]
@marinadistefano @wrightmw here is the screenshot with the new link!
<img width="1032" alt="Screen Shot 2019-07-18 at 9 34 56 AM" src="https://user-images.githubusercontent.com/45801431/61475328-6724a780-a93f-11e9-8493-b19350d32523.png">

**Comment by wrightmw** [comment]
Thanks @markmandell ... I don't think we need an instance, we know what the link looks like. Looks great!

**Comment by wrightmw** [comment]
In R29 release.
